### PR TITLE
Using vectorization to speed up computation

### PR DESCRIPTION
**Pull request by maudzung**

### CODE REVIEW

File: Mapping/rectangle_fitting/rectangle_fitting.py [hunk 1]
```diff
@@ -69,15 +69,11 @@ def _calc_closeness_criterion(self, c1, c2):
...
-        D2 = [min([np.linalg.norm(c2_max - ic2),
-                   np.linalg.norm(ic2 - c2_min)]) for ic2 in c2]
-
-        beta = 0
-        for i, _ in enumerate(D1):
-            d = max(min([D1[i], D2[i]]), self.min_dist_of_closeness_criteria)
-            beta += (1.0 / d)
+        # Vectorization
+        D1 = np.minimum(c1_max - c1, c1 - c1_min)
+        D2 = np.minimum(c2_max - c2, c2 - c2_min)
```
Comment (AtsushiSakai) [inline Mapping/rectangle_fitting/rectangle_fitting.py:74]: Could you please explain why np.linalg.norm is not needed?

**Review by AtsushiSakai — COMMENTED** [review]
Thank you for great PR. You code is more beautiful. I have a question about your change. PTAL.

File: Mapping/rectangle_fitting/rectangle_fitting.py (hunk 1 above)
Comment (maudzung) [reply Mapping/rectangle_fitting/rectangle_fitting.py:74]: `c1_max - c1`, `c1 - c1_min`, `c2_max - c2`, `c2 - c2_min` are 1D vectors, and all components in the 1D vectors are greater or equal 0. Hence we don't need to compute norm values and don't need to take the absolute value also.

Comment (AtsushiSakai) [reply Mapping/rectangle_fitting/rectangle_fitting.py:74]: OK. I understood. Thanks.

**Review by AtsushiSakai — APPROVED** [review]
LGTM. Thank you. 
(I found this script does not have a test, so I created a issue in #436)

**Pull request merged**